<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShpEditor</class>
 <widget class="QWidget" name="ShpEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Редагування перевезення</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QFormLayout" name="formLayout">
     <property name="formAlignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <item row="2" column="0">
      <widget class="QLabel" name="lDspchOfc">
       <property name="text">
        <string>Пункт відправки</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="leDspchOfc">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lArrOfc">
       <property name="text">
        <string>Пункт прибуття</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="leArrOfc">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="lEeWhoGave">
       <property name="text">
        <string>Співробітник,відповідальний за завантаження</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="leEeWhoGave">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="lDateDspch">
       <property name="text">
        <string>Дата та час відправки</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QDateTimeEdit" name="dteDateDspch">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="dateTime">
        <datetime>
         <hour>0</hour>
         <minute>0</minute>
         <second>0</second>
         <year>1752</year>
         <month>9</month>
         <day>14</day>
        </datetime>
       </property>
       <property name="displayFormat">
        <string>dd.MM.yyyy HH:mm</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="lEeWhoTake">
       <property name="text">
        <string>Співробітник,відповідальний за вивантаження</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="leEeWhoTake">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="lDateArr">
       <property name="text">
        <string>Дата та час прибуття</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QDateTimeEdit" name="dteDateArr">
       <property name="readOnly">
        <bool>false</bool>
       </property>
       <property name="dateTime">
        <datetime>
         <hour>0</hour>
         <minute>0</minute>
         <second>0</second>
         <year>1752</year>
         <month>9</month>
         <day>14</day>
        </datetime>
       </property>
       <property name="displayFormat">
        <string>dd.MM.yyyy HH:mm</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="lWaybillNumber">
       <property name="text">
        <string>Номер накладної</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="leWaybillNumber">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lCurSts">
       <property name="text">
        <string>Поточна стадія</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="leCurSts">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_12">
     <item>
      <widget class="QLabel" name="lCurrentEmployer">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbShpArrived">
       <property name="text">
        <string>Відправлення прибули до пункту прибуття</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbAddMailingsToStorage">
       <property name="text">
        <string>Відправлення розвантажені на склад</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="TableViewWithMultipleSelection" name="tvwMailing"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TableViewWithMultipleSelection</class>
   <extends>QTableView</extends>
   <header location="global">tableviewwithmultipleselection.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
